docs/quality.md

```diff
@@ -1,3 +1,60 @@
 # Quality
 
-> @wip
+- Don't want to break this for driver, mongoose
+- Want input from these maintainers on establishing best practies, not to just
+  create more work
+- Result of this will be incorporated into our tooling,
+  [mj](https://github.com/rueckstiess/mj), for automated organization wide
+  checking and scaffolding for new projects
+
+## Testing
+
+- `npm test` returns a non-zero exit code
+- travisci hook
+
+## Code Coverage
+
+- do it if possible
+- tooling will template out istanbul + mocha + coveralls.io
+
+## Documentation
+
+- A lot of different needs here
+- For general small modules, examples in README + comments where it makes sense
+- If you expose more than `module.exports = function(){}`, use jsdoc tags + `dox --api`
+- Tooling will publish a pretty version to github pages for you
+
+## Style
+
+- A `.jshintrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jshintrc)
+- A `.jsfmtrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jsfmtrc)
```

Comment:
@vkarpov15 yep these would be checked in next to package.json.  same idea (here is the style you should follow), but most editor plugins will pickup a local `.jshintrc` and totally miss `package.json` declarations :(

@denniskuczynski jsfmt is what you want your linter to actually do which is just try and fix linting issues instead of just telling you the issue exists.  If you have an editor plugin ([like I do for sublime text](https://github.com/ionutvmi/sublime-jsfmt#jsfmt-for-sublime-text-23)) the amount of time it saves is somewhat absurd...

Again, don't want to dictate style that everyone must follow, just want whatever style rules you have to come along with your repo.  What if this check was just: "Do you have a `./{.jshint*,.jscs*,.editorconfig}` file or `_.has(yourPackageJson, 'jshint') || _.has(yourPackageJson, 'jscs')`?". How does that sound?